### PR TITLE
Created HTML frame for project start and added 'test' style sheet

### PR DESCRIPTION
**Pull request by DijonStamos**
Site isn't looking too pretty at the moment, but all the code was validated successfully. I started trying to style my page but ended up creating some new issues that I had to work backward to solve. :/

### CODE REVIEW

File: blog.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
```
Comment (jimmynono) [inline blog.html:5]: You'll want to add the meta tag to all pages that I referenced week one.

File: blog.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    <title>adopt-a-dog-spring2019-blog</title>
+    <link rel="stylesheet" type="text/css" href="styles/styles.css">
+  </head>
+
+  <body>
+    <img class="headercolor" src="images/adoptadog-logo.svg" alt="adoptadog logo-header">
```
Comment (jimmynono) [inline blog.html]: This isn't in the right spot. The logo should be nested in the header. Also, when creating classes in css you'll want to use kabob casing. Example your html would look similar to

File: blog.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+<header>
+  <h1>Seattle's premiere dog adoption agency</h1>
+  <nav>
+    <a href="index.html">Home</a>
+    <a href="dogs.html">Dogs</a>
+    <a href="blog.html">Blog</a>
+    <a class="checkout" href="checkout.html">Checkout</a>
```
Comment (jimmynono) [inline blog.html:20]: You'll want to make sure to include the button and the check out information

File: blog.html [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+</header>
+    <section>
+      <h2>Travelling with your dog</h2>
+          <p>Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet
+            adis dolenem quidustrum fuga. Faceaquae estioria derum recuptatur,
+            cum volore, undipsa doloreium hillupta aut es ut alitatuscit
+            ommossum haritatur arum qui officae videbiti corporeium faccull
+            oribus es quidignis ipietus explam sus am aut amet ant fugiatum,
+            utem non reptat.
+Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que
```
Comment (jimmynono) [inline blog.html]: Be mindful of indentation.

File: checkout.html [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+          <a href="dogs.html">Dogs</a>
+          <a href="blog.html">Blog</a>
+          <a class="checkout" href="checkout.html">Checkout</a>
+        </nav>
+      </header>
+
+      <h2>Checkout</h2>
+
+    <form method="GET">
+      <label>Name
```
Comment (jimmynono) [inline checkout.html]: You might want to rethink your label and input formatting. 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label

File: dogs.html [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+    <a class="checkout" href="checkout.html">Checkout</a>
+  </nav>
+</header>
+
+    <section>
+      <h2>Here are all of our Dogs!</h2>
+    </section>
+
+    <article>
+      <h3>Murphy</h3>
```
Comment (jimmynono) [inline dogs.html:30]: You're missing images for the dogs. You'll want to dump those in, even if they look terrible without stying

File: dogs.html [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+    <a href="blog.html">Blog</a>
+    <a class="checkout" href="checkout.html">Checkout</a>
+  </nav>
+</header>
+
+    <section>
+      <h2>Here are all of our Dogs!</h2>
+    </section>
+
+    <article>
```
Comment (jimmynono) [inline dogs.html]: I'm not convinced that article is the best element to use as a container of elements.

File: dogs.html [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+      <h3>Murphy</h3>
+        <p>Corrum volorit iandae nimaxim cum restia volor
+          reicid ut et etur sunt arum rendae pla endis re ea erum,
+          qui doluptae</p>
+
+      <h3>Poppy</h3>
```
Comment (jimmynono) [inline dogs.html:38]: You'll want to wrap each dog tile in some sort of wrapping container. It'll make layout much easier

File: dogs.html [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+        <p>Corrum volorit iandae nimaxim cum restia volor
+          reicid ut et etur sunt arum rendae pla endis re ea erum,
+          qui doluptae</p>
+
+      <h3>Jake</h3>
+        <p>Corrum volorit iandae nimaxim cum restia volor
+          reicid ut et etur sunt arum rendae pla endis re ea erum,
+          qui doluptae</p>
+
+      <h3>Angus</h3>
```
Comment (jimmynono) [inline dogs.html:78]: These h3 tags and the p tag should be indented the same level

**Comment by DijonStamos** [comment]
Began styling pages for 'Lesson 02 Assignment 3'. 

CSS is all responsive. I had to do some backtracking to my CSS notes to remind myself how to do some things. I'll continue improving tomorrow evening to try and fix this mess when I'm off work. Any feedback is welcome. Thanks!

**Pull request closed without merging**